{{cookiecutter.package_name}}/Jenkinsfile

```diff
@@ -28,6 +28,9 @@ Note that updating the shared repo will take affect on the next pipeline invocat
 library("vivarium_build_utils@${get_vbu_version()}")
 
 reusable_pipeline(
+    // Add additional branches for cron jobs below.
+    // Note: Branches not named "main" will require a manual trigger of the first build 
+    // for subsequent scheduled builds to run.
```

Comment:
This might be easier to not miss?

```
reusable_pipeline(
    scheduled_branches: [
        // NOTE: you must manually trigger the first build when adding new branches here!
        'main',
    ],
    skip_doc_build: true,
    upstream_repos: ['vivarium', 'vivarium_public_health', 'gbd_mapping', 'vivarium_cluster_tools', 'vivarium_inputs'],
    ...
```